### ISSUE TITLE
DATABASES NOT WORKING

### ISSUE THREAD

**Issue by KillerPaperMice** [Whiskr/whiskr#125]
## tl;dr
I can't log in to any recently created user and my new matches aren't saving for any ~~existing~~ oAuth users (my gmail ~~and catlady~~ Catlady is working). 

## Wait, What's Going on Exactly?
If I try to Login with a newly created account I get an incorrect password error (in Redux dev tools). 
I double checked multiple times to make sure it was the right password so I was thinking it's really because the user doesn't get saved **BUT** every time I make a new user the Id number does increment to the right one. So, maybe there is something else going on?

Also with the pet matches I made sure it is still putting the userId and pet Id correctly when I reviewed it in the Store but when I log out and Log back in again it doesn't load any of the new matches I've made from the last few days. ~~So back to maybe Database issues?~~

*Update dec 11, 2017*
New users are definitely being stored in the database and users can be updated. The Error message is "wrong password"  not  "user not found". So I think the problem is how the passwords are being encrypted. 

The Match problem might be an issue where the matches are added to the store but not the database but this only happens with oAuth users. So maybe those aren't being linked properly? It's weird though because users have a set id that is being used for the matches so I don't know why it's not saving.
